File: WCF/Windows SDK WCF Samples/TechnologySamples Beta 1/Extensibility/Binding/WSStreamedHttpBinding/VB/WSStreamedHttpBinding/StreamedTransferMode.vb

' Copyright (c) Microsoft Corporation.  All rights reserved.

Imports System.ServiceModel
Imports System.ServiceModel.Channels
Namespace Microsoft.ServiceModel.Samples
    Public Enum StreamedTransferMode
        Streamed = TransferMode.Streamed
        StreamedRequest = TransferMode.StreamedRequest
        StreamedResponse = TransferMode.StreamedResponse
    End Enum
End Namespace

File: WCF/Windows SDK WCF Samples/TechnologySamples Beta 1/Extensibility/Binding/WSStreamedHttpBinding/CS/WSStreamedHttpBinding/StreamedTransferMode.cs
//------------------------------------------------------------------------------
// Copyright (c) Microsoft Corporation.  All rights reserved.
//------------------------------------------------------------------------------
namespace Microsoft.ServiceModel.Samples
{
    using System.ServiceModel;
	using System.ServiceModel.Channels;

    public enum StreamedTransferMode
    {
        Streamed = TransferMode.Streamed,
        StreamedRequest = TransferMode.StreamedRequest,
        StreamedResponse = TransferMode.StreamedResponse
    }
}
